sale_order_invoicing_queued/readme/USAGE.rst

```diff
@@ -0,0 +1,9 @@
+#. Go to *Sales > Invoicing > Orders to Invoice*.
+#. Select sales orders whose invoicing you want to enqueue.
+#. Click on *Action > Invoice Order*.
+#. Click on "Enqueue Invoicing" button.
+#. On that moment, jobs will be enqueued on foreground, but the control will
```

Comment:
```suggestion
#. On that moment, jobs will be enqueued on the background, but the control will
```

Reply:
No, they are enqueued on foreground, and executed on background.